### ISSUE TITLE
Nativefier should better validate arguments

### ISSUE THREAD

**Issue by ObiDriftKenobi** [jiahaog/nativefier#696]
### Description
I tried to nativefy the website aminoapps.com with the flags: internal urls: "*." (all links will open in the app) and hide window frame. The command is working, I did this with youtube a month ago and it worked perfectly
### Steps to reproduce issue
1: nativefier aminoapps.com -internal-urls "*." -hide-window-frame
### Details

- Are you nativefying a *public* website? Yes
- Full nativefier command used to build your app: `nativefier aminoapps.com -internal-urls "*." -hide-window-frame`
- Version of Nativefier (run `nativefier --version`): `v7.6.7`
- Version of node.js (run `node --version`): `v8.11.3`
- OS: `Windows 10 64bit x84`
- Error message / stack trace (if any):

```
Downloading tmp-1944-0-electron-v-r-win32-x64.zip
Error: GET https://github.com/electron/electron/releases/download/v-r/electron-v-r-win32-x64.zip returned 404
Error: Failed to find Electron v-r for win32-x64 at https://github.com/electron/electron/releases/download/v-r/electron-v-r-win32-x64.zip
    at Request.<anonymous> (C:\Users\danie\AppData\Roaming\npm\node_modules\nativefier\node_modules\nugget\index.js:169:61)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at Request.onRequestResponse (C:\Users\danie\AppData\Roaming\npm\node_modules\nativefier\node_modules\request\request.js:1066:10)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at HTTPParser.parserOnIncomingClient (_http_client.js:543:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:112:17)
    at TLSSocket.socketOnData (_http_client.js:440:20)
    at emitOne (events.js:116:13)
    at TLSSocket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at TLSSocket.Readable.push (_stream_readable.js:208:10)
    at TLSWrap.onread (net.js:597:20)
```

**Comment by ronjouch** [comment]
@PancakeGD `nativefier --help` says it's used like `nativefier [options] <targetUrl>`, not `nativefier <targetUrl> [options]`

So, `nativefier --internal-urls '*.' --hide-window-frame aminoapps.com`, not `nativefier aminoapps.com -internal-urls "*." -hide-window-frame`

Nativefier could/should do a better job at detecting bad options like these, and complaining rather than crashing (in a weird way). Renaming the issue to suggest this. @PancakeGD / passersby, PR welcome 🙂.

**Comment by ronjouch** [comment]
This is fixed in 8.0.4: with the above-mentioned example, Nativefier now shows the help page instead of crashing. Closing.

**Comment by ObiDriftKenobi** [comment]
Damn, this is an old issue.
